.github/workflows/test.yml

```diff
@@ -138,23 +138,28 @@ jobs:
             !**.md
             !**.mdx
 
-      - name: Go related changes
-        id: go
+      - name: Turborepo Go related changes
+        id: turborepo_go
         uses: technote-space/get-diff-action@v6
         with:
           PATTERNS: |
             cli/**
 
-      - name: Go E2E related changes
-        id: go_e2e
+      - name: Turborepo integration tests changes
+        id: turborepo_integration
         uses: technote-space/get-diff-action@v6
         with:
           PATTERNS: |
-            cli/**
-            crates/turborepo*
-            crates/turborepo*/**
-            crates/turbo-updater
```

Comment:
These crates patterns were incomplete, so I'm using the existing `turborepo_rust` definition above to determine when to run integration and e2e tests